File: 프로그래머스/unrated/181853. 뒤에서 5등까지/뒤에서 5등까지.java
import java.util.Arrays;
class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = {};
        if(num_list.length<5){
            answer = num_list.clone();
        }else{
            answer = new int[5];
            Arrays.sort(num_list);
            for(int i=0; i<answer.length; i++){
              answer[i] = num_list[i];
            }
            
        }
        return answer;
    }
}

File: 프로그래머스/0/181853. 뒤에서 5등까지/뒤에서 5등까지.js
function solution(num_list) {
    // num_list를 오름차순으로 정렬
    num_list.sort((a, b) => a - b);
    
    // 정렬된 리스트의 첫 5개의 요소를 슬라이스하여 반환
    let answer = num_list.slice(0, 5);
    
    return answer;
}